.github/resources/scripts/deploy-kfp.sh

```
@@ -24,47 +24,68 @@ C_DIR="${BASH_SOURCE%/*}"
+      USE_PROXY=true
+      shift
+      ;;
+    --cache-disabled)
+      CACHE_DISABLED=true
+      shift
+      ;;
+  esac
 done
 
-shift $((OPTIND-1))
+if [ "${USE_PROXY}" == "true" && "${PIPELINES_STORE}" == "kubernetes" ]; then
```

Shellcheck (0.10.0)

[error] Instead of [ a && b ], use [ a ] && [ b ].

(SC2107)